wxt_cidrbot/cidrbot.py

```diff
@@ -26,67 +28,134 @@ def __init__(self):
+            self.wxt_access_token = os.getenv("WEBEX_TEAMS_ACCESS_TOKEN")
+        else:
+            logging.error("Environment variable WEBEX_TEAMS_ACCESS_TOKEN must be set")
+            sys.exit(1)
+
         # Initialize Api
         self.Api = WebexTeamsAPI()
 
+        # Init sibling py files
         self.get_command = cmd_list.cmdlist()
         self.git_handle = git_api_handler.githandler()
+        self.dynamo = dynamo_api_handler.dynamoapi()
+        self.webex = webex_edit_message.webex_message()
 
     def send_wbx_msg(self, room, message, pt_id):
         self.Api.messages.create(room, markdown=message, parentId=pt_id)
 
+    def send_directwbx_msg(self, person_id, message):
+        self.Api.messages.create(toPersonId=person_id, markdown=message)
+
+    # Send a daily message to the cidrbot users chatroom: Every day at 11am est: Cron expression 0 15 * * ? *
     def send_timed_msg(self):
-        message = f"**Daily Issues:**\n" + self.git_handle.issues_list("List")
+        message = self.git_handle.scan_repos("List", 'All', self.dynamo.dynamo_db("repos", None, None, None), False)
         self.send_wbx_msg(self.cidrbot_room_id, message, None)
 
+    # Send a message to all users with reminders enabled: Every monday at 12pm est: Cron expression 0 16 ? * 2 *
+    def weekly_reminder_email(self):
+        remind_users = dict(self.dynamo.dynamo_db('all_users', None, None, None))
+        assigned_issues_dict = self.git_handle.scan_repos(
+            "Dict", 'All', self.dynamo.dynamo_db("repos", None, None, None), False
+        )
+
+        for i in remind_users['Items']:
+            user_name = i['User']
+            person_id = i['person_id']
+            self.logging.debug(user_name + " " + person_id)
+
+            message = self.get_command.get_user_issues(assigned_issues_dict, user_name)
+            if message != "No issues":
+                self.send_directwbx_msg(person_id, message)
+
+    # Process webhook request from lambda_function
     def webhook_request(self, event):
         json_string = json.loads((event["body"]))
+        webex_msg_sender = json_string['data']['personEmail']
         event_type = json_string['name']
+        user_id = json_string['data']['personId']
 
         if event_type == "New user":
-            text = self.get_command.new_user(json_string)
+            user_name = self.Api.people.get(user_id).firstName
+            text = self.get_command.new_user(json_string, webex_msg_sender, user_name)
             self.send_wbx_msg(self.cidrbot_room_id, text, None)
-        else:
-            self.message_event(json_string, event_type)
-
-    def message_event(self, json_string, event_type):
-        webex_msg_sender = json_string['data']['personEmail']
+        elif event_type == "User left":
+            self.logging.debug("User left")
+            self.dynamo.dynamo_db('delete_user', webex_msg_sender, None, None)
+        elif webex_msg_sender != "CIDRBot@webex.bot":
```

Comment:
Is this something that would need to change if say we installed another instance of the bot elsewhere?